tests/data/region_processing/external_repo_test/nomenclature.yaml

```diff
@@ -7,8 +7,11 @@ repositories:
     hash: cb85704
 definitions:
   region:
-    repository: common-definitions
+    repository:
+      name: common-definitions
   variable:
-    repository: common-definitions
+    repository:
```

Comment:
Would it make sense to keep the short-hand notation here (without the *name* attribute) to make sure that this implementation continues to work? (Please add a comment if you revert).

Reply:
Good point, I restored the original notation on all other files but missed that one. It's fixed now.